appeals-service-api/src/routes/appeals.js

```diff
@@ -3,14 +3,21 @@ const appealsController = require('../controllers/appeals');
 
 const routes = new Router();
 
-routes.route('/').post(appealsController.create).get(appealsController.list);
+routes
+  .route('/')
+  .post(appealsController.create)
+  .get(appealsController.list);
 
 routes
   .route('/:uuid')
   .delete(appealsController.delete)
   .get(appealsController.get)
   .put(appealsController.update);
 
+routes
+  .route('/files/upload')
+  .post(appealsController.uploadFile)
```

Comment:
The appeals service has no knowledge of uploading - this should be done in the Document Service API